### ISSUE TITLE
Norde-Red Won't Run After Update 17.0.13

### ISSUE THREAD

**Issue by NeuzNL** [hassio-addons/addon-node-red#1891]
# Problem
After update to node-red 17.0.13 it won't start anymore.

Core
2024.6.1

Supervisor
2024.06.0

Operating System
12.3

Frontend
20240605.0

# Error message
```
s6-rc-compile: fatal: invalid /package/admin/s6-overlay-3.1.6.2/etc/s6-rc/sources/base-addon-banner/type: must be oneshot, longrun, or bundle
s6-rc: fatal: unable to take locks: No such file or directory
s6-linux-init-shutdownd: warning: /run/s6/basedir/scripts/rc.shutdown exited 111
```

# Tried Solution
I have already tried this solution but no joy
https://github.com/hassio-addons/addon-node-red/issues/1791

Can someone help me out?

**Comment by NeuzNL** [comment]
Solution:
I have deleted Node-Red in total and reinstall. Only minor thing you have to make all flows new.

**Comment by lightmaster** [comment]
Don't think this should be closed, as Node-RED 17.0.13 is failing to start for me after updating as well.

```
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service nginx: stopping
[21:24:45] INFO: Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service init-nginx: stopping
s6-rc: info: service nodered: stopping
s6-rc: info: service init-nginx successfully stopped
10 Jun 21:24:45 - [info] Stopping flows
10 Jun 21:24:45 - [info] [server:Home Assistant] Closing connection to http://supervisor/core
10 Jun 21:24:45 - [info] [mqtt-broker:MQTT] Disconnected from broker: mqtt://emqx:1883
10 Jun 21:24:45 - [info] Stopped flows
[21:24:45] INFO: Service Node-RED exited with code 0 (by signal 0)
s6-rc: info: service nodered successfully stopped
s6-rc: info: service init-nodered: stopping
s6-rc: info: service init-nodered successfully stopped
s6-rc: info: service init-customizations: stopping
s6-rc: info: service init-customizations successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service base-addon-log-level: stopping
s6-rc: info: service base-addon-log-level successfully stopped
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service base-addon-banner: stopping
s6-rc: info: service base-addon-banner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
exec /init: exec format error
```
